--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -416,11 +416,11 @@ install(TARGETS ${name}
         LIBRARY DESTINATION ${LIB_INSTALL_DIR}
         )
 install( FILES ${HFILES} DESTINATION ${INCLUDE_INSTALL_DIR} )
-if(MSVC)
-    # install(FILES $<TARGET_PDB_FILE:${name}> DESTINATION lib OPTIONAL)
-    INSTALL(FILES ${PROJECT_BINARY_DIR}/${name}.dir/Debug/${name}.pdb
-    DESTINATION lib CONFIGURATIONS Debug )
-endif()
+#if(MSVC)
+#    # install(FILES $<TARGET_PDB_FILE:${name}> DESTINATION lib OPTIONAL)
+#    INSTALL(FILES ${PROJECT_BINARY_DIR}/${name}.dir/Debug/${name}.pdb
+#    DESTINATION lib CONFIGURATIONS Debug )
+#endif()
 
 #------------------------------------------------------------------------
 # Dynamic Library
