--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -686,18 +686,4 @@ endif ()
 include(CPack)
 
 
-#------------------------------------------------------------------------
-# pkg-config
-#------------------------------------------------------------------------
-configure_file(
-    "${CMAKE_CURRENT_SOURCE_DIR}/${LIB_NAME}.pc.cmake.in"
-    "${CMAKE_CURRENT_BINARY_DIR}/${LIB_NAME}.pc"
-    @ONLY
-    )
-    
-install(FILES
-    "${CMAKE_CURRENT_BINARY_DIR}/${LIB_NAME}.pc"
-    DESTINATION "${LIB_INSTALL_DIR}/pkgconfig"
-    )
-
 # eof
